infra/analytics/app-config/env-config/environment-variables.tf

```diff
@@ -61,6 +61,8 @@ locals {
     #     -- the "app" database is where the analytics data is stored
     #     GRANT ALL PRIVILEGES ON DATABASE app TO metabaseuser;
     #     GRANT CONNECT ON DATABASE app TO metabaseuser;
+    #     GRANT USAGE ON SCHEMA app TO metabaseuser;
+    #     GRANT SELECT ON ALL TABLES IN SCHEMA app TO metabaseuser;
```

Comment:
One thing we might additionally need is to alter the default permissions.

One thing that trips up permissions on Postgres is that `GRANT X ..` generally only changes permissions for things that already exist - although I still don't 100% follow the logic on when that applies.

It can't hurt to put:
```sql
ALTER DEFAULT PRIVILEGES GRANT USAGE ON TABLES TO metabaseuser;
```